WebRoot/js/zimbraMail/briefcase/view/ZmPreviewPaneView.js

```diff
@@ -385,6 +399,19 @@ ZmPreviewPaneView.prototype._listSelectionListener = function(ev, item) {
     	} else {
     		this._previewView.set(item);
     	}
+    } else if(!this._controller.isReadingPaneOn() && item && item.folderId == ZmFolder.ID_FILE_SHARED_WITH_ME){
+        var restUrl = item.getRestUrl();
+        restUrl = AjxStringUtil.fixCrossDomainReference(restUrl);
+        var req = new XMLHttpRequest();
+        req.open("GET", restUrl, false);
+        req.send();
+        var status = req.status;
+        if(status == 404){
+            var listId = this._controller._listView[this._controller.getCurrentViewId()]._getFieldId(item,ZmItem.F_NAME);
+            var name = document.getElementById(listId);
+            name.innerHTML = "<del>"+ name.innerHTML +"<del>";
```

Comment:
innerHtml maybe unsafe from security point of view

Reply:
replaced it with css  class.